### PR TITLE
ABM create agents

### PR DESCRIPTION
**Pull request by danyal1234**
Create agents by parsing CSV and using census probabilities to give attributes

### CODE REVIEW

File: .vscode/c_cpp_properties.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+    "configurations": [
```
Comment (danyal1234) [inline .vscode/c_cpp_properties.json:2]: @JamesBusch what is this my mac created and will it break master?

File: src/agent.cpp [hunk 2]
```diff
@@ -11,6 +11,127 @@
 #include "agent.hh"
 
 // Constructor
-Agent::Agent() {
-    //
+Agent::Agent(AgentInfo agentInfo) {
```
Comment (danyal1234) [inline src/agent.cpp:14]: add comments to this file tmrrw

File: include/agent.hh [hunk 3]
```diff
@@ -12,21 +12,35 @@
...
 
 class Location;
 //Declare simulation class
 class Agent{
     private:
-    AgeGroup age;
+    AgentInfo info;
     HouseholdIncome income;
     string ethnicity;
-    string gender;
```
Comment (danyal1234) [inline include/agent.hh:29]: remove not needed anymore

File: include/agent.hh [hunk 4]
```diff
@@ -12,21 +12,35 @@
...
     string maritalStatus;
     Location* address;
     string education;
```
Comment (danyal1234) [inline include/agent.hh:46]: forgot about this smh i will look into the data tmrrw

File: .vscode/c_cpp_properties.json (hunk 1 above)
Comment (JamesBusch) [reply .vscode/c_cpp_properties.json:2]: No its just config files for vscode, it has nothing to do with compilation or run time

**Pull request merged**